### PR TITLE
Update readme text

### PR DESCRIPTION
**Pull request by SirPryderi**

### CODE REVIEW

File: TUTORIAL-ASSETS.md [hunk 1]
```diff
@@ -81,20 +81,19 @@ Once you're ready, export your layer as a GeoJSON file the same way you did duri
...
-Importing Assets from an AutoCAD DXF file is a little more involved that the GeoJSON submission method. 
+Importing Assets from an AutoCAD DXF file is a little more involved that the GeoJSON submission method.
 
-For this example, we'll be using a simple 'office' CAD file that looks like the image below.  We'll specifically be mapping the rectangular section highlighted below to our new Indoor Map's floor - it's not an exact match, but for the purposes of this example it'll demonstrate how to read the desk positions correctly.
+For this example, we'll be using a simple 'office' CAD file that looks like the image below. We'll specifically be mapping the rectangular section highlighted below to our new Indoor Map's floor - it's not an exact match, but for the purposes of this example it'll demonstrate how to read the desk positions correctly.
 
 ![Image showing the mapped area of the CAD](images/cadtutorial/area.png)
 
 To do this, we need the following:
-* An AutoCAD DXF file with named blocks identifying the soft furnishings we wish to extract (e.g. desks).
```
Comment (iainbethune) [inline TUTORIAL-ASSETS.md:97]: Replace "soft furnishings" with "assets".

File: TUTORIAL-ASSETS.md [hunk 2]
```diff
@@ -110,22 +109,21 @@ mapX,mapY,pixelX,pixelY,enable
...
 ```
 
-This describes four points - the first two coordinates of each line specify X and Y coordinates in the DXF file, while the next two coordinates are a longitude and latitude.  Looking at our example DXF file and floorplan GeoJSON, the 4 points refer to the following:
+This describes four points - the first two coordinates of each line specify X and Y coordinates in the DXF file, while the next two coordinates are a longitude and latitude.  Looking at our example DXF file and floor plan GeoJSON, the 4 points refer to the following:
 
 ![Image showing the positions of the 4 georeference coordinates](images/cadtutorial/georeferencing.png)
 
-> ## NOTE 
->> You can add additional points as necessary - generally the more points you map, the more accurate the extraction process will be.  For a small,  rectangular floorplan like this, 4 are enough!
+> :information_source: You can add additional points as necessary - generally the more points you map, the more accurate the extraction process will be. For a
```
Comment (iainbethune) [inline TUTORIAL-ASSETS.md:116]: Sentence got truncated here.

File: TUTORIAL-ASSETS.md [hunk 3]
```diff
@@ -110,22 +109,21 @@ mapX,mapY,pixelX,pixelY,enable
...
+> :information_source: You can add additional points as necessary - generally the more points you map, the more accurate the extraction process will be. For a 
 
 ### Creating the config.json file
 
 In our example, we'll be extracting the highlighted desks and representing them with a similar 3D model:
 
 ![Image showing the desks we'll be extracting from the CAD](images/cadtutorial/desks_cad.png)
 
-The config.json file part of our submission details how the CAD will be read by the extraction process - essentially telling it which parts are desks, chairs etc. along with how to determine their orientation and what 3D Model to use to represent the Asset in your map.  Our example config file looks like this:
+The config.json file part of our submission details how the CAD will be read by the extraction process - essentially telling it which parts are desks, chairs etc. along with how to determine their orientation and what 3D Model to use to represent the Asset in your map. Our example config file looks like this:
```
Comment (iainbethune) [inline TUTORIAL-ASSETS.md:124]: lower case "model". Generally Asset is capitalised and "3D model" is lower case which I think is fine.

File: TUTORIAL-ASSETS.md [hunk 4]
```diff
@@ -247,40 +245,33 @@ Hit continue and after a bit more processing you should now see your Assets insi
 
 ## Editing your Assets
 
-Now that you've imported your Assets, you can make any changes you wish to their position or orientation. To do this, you can either click on the model directly, or select from the Asset List panel by selecting the Asset Category.
-
-![Image showing the Asset Category List](images/cadtutorial/asset_list_category.png)
-
-Clicking on this will show you your list of Assets of the specified category. Here you can select them, which will move the camera and highlight the selected Asset, and also delete individual ones as required.
+Now that you've imported your Assets, you can make any changes you wish to their position or orientation. To do this, you can either click on the model directly, or select it from the Asset List panel. In the Asset List panel you can select your Assets, which will highlight the selected one, and also delete individual ones as required.
```
Comment (iainbethune) [inline TUTORIAL-ASSETS.md:248]: Replace "In the Asset List panel you can select your Assets, which will highlight the selected one, and also delete individual ones as required." with "Selected Assets are highlighted onscreen.  In the Asset List panel you can also delete individual assets if required."

File: TUTORIAL-ASSETS.md [hunk 5]
```diff
@@ -247,40 +245,33 @@ Hit continue and after a bit more processing you should now see your Assets insi
...
 
 ![Image showing the Asset Category List](images/cadtutorial/asset_list_highlight.png)
 
-You can also click and drag selected Assets by clicking the ![2D Edit](images/cadtutorial/2d_edit.png) button and clicking and dragging the gizmo that appears.
+You can also click and drag selected Assets by clicking the dragging the gizmo.
 
 ![Image showing desk movement](images/cadtutorial/desk_move.gif)
 
 Selecting an Asset also shows the Asset Properties panel, which allows you to edit the selected Asset in more detail.
```
Comment (iainbethune) [inline TUTORIAL-ASSETS.md:256]: Suggest moving the Asset Properties section before the click and drag section, as it reads better that way round.

File: TUTORIAL-ASSETS.md [hunk 6]
```diff
@@ -247,40 +245,33 @@ Hit continue and after a bit more processing you should now see your Assets insi
...
-
-![Image showing the Asset Category List](images/cadtutorial/asset_list_category.png)
-
-Clicking on this will show you your list of Assets of the specified category. Here you can select them, which will move the camera and highlight the selected Asset, and also delete individual ones as required.
+Now that you've imported your Assets, you can make any changes you wish to their position or orientation. To do this, you can either click on the model directly, or select it from the Asset List panel. In the Asset List panel you can select your Assets, which will highlight the selected one, and also delete individual ones as required.
 
 ![Image showing the Asset Category List](images/cadtutorial/asset_list_highlight.png)
 
-You can also click and drag selected Assets by clicking the ![2D Edit](images/cadtutorial/2d_edit.png) button and clicking and dragging the gizmo that appears.
+You can also click and drag selected Assets by clicking the dragging the gizmo.
```
Comment (iainbethune) [inline TUTORIAL-ASSETS.md:252]: "You can also move selected Assets by clicking and dragging the gizmo."

File: TUTORIAL-ASSETS.md [hunk 7]
```diff
@@ -220,15 +218,15 @@ In this above example, we specify a layer of the CAD named `labelLayer` to read
...
 
-## Importing
+## Importing your Assets
 
 With your files prepared, you can now begin the extraction process!
 
 - If you're importing GeoJSON directly, you're all set!
 - If you're importing the combined cad.dxf, config.json and georeference.points files, begin by zipping the 3 files together in a .zip file (similar to cad_submission.zip in our examples directory). 
 - Navigate to the [WRLD Indoor Map Designer](https://mapdesigner.wrld3d.com/indoormap/latest/) (login if necessary).
 - Select your Indoor Map from the drop down list and go inside by clicking the Entrance Marker on the map.
```
Comment (iainbethune) [inline TUTORIAL-ASSETS.md:228]: Add in a step "Select the floor you wish to add Assets to using the elevator widget on the right of the screen".  We make reference later to the fact that asset sets are associated with floors, so we should make this obvious up front that the user has to choose a floor before importing.

File: TUTORIAL-ASSETS.md (hunk 2 above)
Comment (SirPryderi) [reply TUTORIAL-ASSETS.md:116]: Whoopsie! 😅

**Pull request merged**